src/ONYXKEYS.ts

```diff
@@ -379,12 +378,85 @@ const ONYXKEYS = {
     },
 } as const;
 
-type OnyxKeysMap = typeof ONYXKEYS;
-type OnyxCollectionKey = ValueOf<OnyxKeysMap['COLLECTION']>;
-type OnyxKey = DeepValueOf<Omit<OnyxKeysMap, 'COLLECTION'>>;
-type OnyxFormKey = ValueOf<OnyxKeysMap['FORMS']>;
+type AllOnyxKeys = DeepValueOf<typeof ONYXKEYS>;
+
+type OnyxFormValuesMapping = {
+    [ONYXKEYS.FORMS.ADD_DEBIT_CARD_FORM]: FormTypes.AddDebitCardForm;
+    [ONYXKEYS.FORMS.WORKSPACE_SETTINGS_FORM]: FormTypes.WorkspaceSettingsForm;
+    [ONYXKEYS.FORMS.WORKSPACE_RATE_AND_UNIT_FORM]: FormTypes.WorkspaceRateAndUnitForm;
+    [ONYXKEYS.FORMS.CLOSE_ACCOUNT_FORM]: FormTypes.CloseAccountForm;
+    [ONYXKEYS.FORMS.PROFILE_SETTINGS_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.DISPLAY_NAME_FORM]: FormTypes.DisplayNameForm;
+    [ONYXKEYS.FORMS.ROOM_NAME_FORM]: FormTypes.RoomNameForm;
+    [ONYXKEYS.FORMS.REPORT_DESCRIPTION_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.LEGAL_NAME_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.WORKSPACE_INVITE_MESSAGE_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.DATE_OF_BIRTH_FORM]: FormTypes.DateOfBirthForm;
+    [ONYXKEYS.FORMS.HOME_ADDRESS_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.NEW_ROOM_FORM]: FormTypes.NewRoomForm;
+    [ONYXKEYS.FORMS.ROOM_SETTINGS_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.NEW_TASK_FORM]: FormTypes.Form;
+    [ONYXKEYS.FORMS.EDIT_TASK_FORM]: FormTypes.Form;
```

Comment:
Sorry for not seeing this earlier, but why are there all these types that are just `FormTypes.Form`, even though we have more specific types for these forms, such as the one defined in `src/types/form/EditTaskForm` ?

Reply:
I think it was adjusted at some point and I made a mistake while resolving conflicts. I'll fix it in a follow up 👍